bootstrap.sh

```diff
@@ -2,10 +2,21 @@
 
 apt-get update
 apt-get install python3 python3-pip -y
-pip3 install flask Flask-SQLAlchemy SQLAlchemy pymysql
+pip3 install -r requirments.txt
+# pip3 install flask Flask-SQLAlchemy SQLAlchemy pymysql Flask-WTF Flask-Bootstrap Flask_login pytest
```

Comment:
please remove the comment.
comments should be here to explain code, not for code you decided to erase